### PR TITLE
Suggested improvement to make intended workflow more clear

### PR DESCRIPTION
**Pull request by dwk2**
I am suggesting this improvement to Step 3 on the [install-local-migrate](https://islandora-collaboration-group.github.io/ISLE/install/install-local-migrate/#step-3-git-clone-the-production-islandora-drupal-site-code) docs page. 

This PR puts the two required repositories (`mysite-islandora` and `mysite-isle`) next to each other, instead of nesting the `mysite-islandora` within the `mysite-isle`'s `data/apache/html` folder.

I think separating these two repos makes it easier to:
- understand bind-mounts
- blow up and re-clone either repo, if necessary, w/o the overhead of having to remember to reclone an interior nested repo as well.

Ultimately, developers can do -- or nest -- as they wish, however, I think that recommending the nested route adds complexity, when our real job here is to convey the idea of having a similar workflow that extends from Local to Staging to Production.

I recognize that, if this PR is accepted, I will also need to update related text within the `install-local-new` doc.

I invite others to share your thoughts and opinions.

### CODE REVIEW

**Comment by McFateM** [comment]
I haven't looked at the details of this change, but the description makes sense.  I've built an ISLE v1.2.0 "dg-isle" repo and am working on the same version of "dg-islandora" and look forward to continuing that work with this PR in place for guidance.

**Comment by bookishgirl** [comment]
It sounds like a logical and useful change to me.

File: docs/install/install-local-migrate.md [hunk 1]
```diff
@@ -231,17 +231,23 @@ Continue the local setup as directed below and ultimately import some objects fr
...
+* While you may create your new bind-mount directory anywhere, we suggest that you put it at the same level as your existing ISLE directory.
+* `cd ..`
+* `git clone https://yourgitproviderhere.com/yourinstitutionhere/yourprojectnamehere-islandora.git`
+    * Example: The above process created a folder named "yourprojectnamehere-islandora"
+* Open `docker-compose.local.yml`
+    * Search for "apache:"
+    * Find the sub-section called "volumes:"
+    * Find this line: "- ./data/apache/html:/var/www/html:cached"
+    * Edit the above line to be like this: ` - ../yourprojectnamehere-islandora:/var/www/html:cached`
+* Your bind-mount is now complete.
```
Comment (bseeger) [inline docs/install/install-local-migrate.md]: I'd recommend saying "Your bind-mount setup is now complete."

Comment (bseeger) [inline docs/install/install-local-migrate.md]: I support this change and think it's a good idea.  I think that some of the wording could change a little.  Some suggestions below:
---

## Step 3: Git Clone the Production Islandora Drupal Site Code

This step assumes you have an existing Islandora Drupal site checked into a git repository. (If not, then you'll need to check your Drupal site into a git repository following the same commands from [Local ISLE Installation: New Site](../install/install-local-new.md) documentation.)

**Note:** If below you see a "fatal: Could not read from remote repository." error, then please read [Fatal: Could not read from remote repository](../install/install-troubleshooting.md#fatal-could-not-read-from-remote-repository).

Using the still open `terminal` (Windows: `PowerShell`):

* Create a location outside of your ISLE directory where your Islandora Drupal site code will be stored.
  While you may create this location anywhere, we suggest that you put it at the same level as your existing ISLE directory.
    * from your ISLE directory, `cd ..`
    * `git clone https://yourgitproviderhere.com/yourinstitutionhere/yourprojectnamehere-islandora.git`
        * Example: The above process created a folder named "yourprojectnamehere-islandora"
* Now create a bind-mount to this location in the `docker-compose.local.yml` in the ISLE repository:
    * Search for "apache:"
    * Find the sub-section called "volumes:"
    * Find this line: `- ./data/apache/html:/var/www/html:cached`
    * Edit the above line to be like this: ` - ../yourprojectnamehere-islandora:/var/www/html:cached`
* Your Production Islandora site code is now configured to be used in this local setup.`

Comment (bseeger) [reply docs/install/install-local-migrate.md]: ## Step 3: Git Clone the Production Islandora Drupal Site Code

This step assumes you have an existing Islandora Drupal site checked into a git repository. (If not, then you'll need to check your Drupal site into a git repository following the same commands from [Local ISLE Installation: New Site](../install/install-local-new.md) documentation.)

**Note:** If below you see a "fatal: Could not read from remote repository." error, then please read [Fatal: Could not read from remote repository](../install/install-troubleshooting.md#fatal-could-not-read-from-remote-repository).

Using the still open `terminal` (Windows: `PowerShell`):

* Create a location outside of your ISLE directory where your Islandora Drupal site code will be stored.
  While you may create this location anywhere, we suggest that you put it at the same level as your existing ISLE directory.
    * from your ISLE directory, `cd ..`
    * `git clone https://yourgitproviderhere.com/yourinstitutionhere/yourprojectnamehere-islandora.git`
        * Example: The above process created a folder named "yourprojectnamehere-islandora"
* Now create a bind-mount to this location in the `docker-compose.local.yml` in the ISLE repository:
    * Search for "apache:"
    * Find the sub-section called "volumes:"
    * Find this line: `- ./data/apache/html:/var/www/html:cached`
    * Edit the above line to be like this: ` - ../yourprojectnamehere-islandora:/var/www/html:cached`
* Your Production Islandora site code is now configured to be used in this local setup.

Comment (dwk2) [reply docs/install/install-local-migrate.md]: excellent clarifications @bseeger - thank you! I have inserted these edits to resolve the steps for creating a local bind-mount.

**Pull request merged**